nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -41,6 +42,10 @@
 
     String EVENT_NAME = "download";
 
+    String EVENT_NAME_FROM_DOC = "downloadFromDoc";
+
+    String INFO = "info";
```

Comment:
No need for these two to be in the interface, they're just internal markers for the `getDownloadPathAndAction` action. I'd prefer if we used an internal enum for the action (right part) actually it would be cleaner.